examples/render_grid/run.py

```diff
@@ -0,0 +1,334 @@
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+# Before running this example, you should have a good understanding of the example in render_image/run.py
+
+import math
+import os
+
+import cv2
+import numpy as np
+
+import spear
+
+if __name__ == "__main__":
+
+    # define the number of cameras in the grid
+    rows = 3
+    cols = 3
+
+    # create instance
+    config = spear.get_config(
```

Comment:
Prefer single line if less than 160 chars wide. Slightly over this limit is ok if there is a good aesthetic reason:

```
    config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
```